superset-frontend/plugins/plugin-chart-echarts/src/MixedTimeseries/controlPanel.tsx

```diff
@@ -253,11 +255,33 @@ function createCustomizeSection(
   ];
 }
 
+function createAdvancedAnalyticsSection(
+  label: string,
+  controlSuffix: string,
+): ControlPanelSectionConfig {
+  const aaWithSuffix = cloneDeep(sections.advancedAnalyticsControls);
+  aaWithSuffix.label = label;
+  if (!controlSuffix) {
+    return aaWithSuffix;
+  }
+  aaWithSuffix.controlSetRows.forEach(row =>
+    row.forEach((control: CustomControlItem) => {
+      if (control?.name) {
+        // eslint-disable-next-line no-param-reassign
+        control.name = `${control.name}${controlSuffix}`;
+      }
+    }),
+  );
+  return aaWithSuffix;
+}
+
```

Comment:
One minor observation: the "Periods" sets the visibility based on the state of the "Rolling Type" control:
<img width="537" alt="image" src="https://user-images.githubusercontent.com/33317356/165491265-6b139d39-2e20-4e85-9472-daef5ec0bab3.png">
This causes the "Periods" section in the B section to have an incorrect visibility (should be hidden for `cumsum`):
<img width="316" alt="image" src="https://user-images.githubusercontent.com/33317356/165490846-4eb22cb0-11cf-449c-b00a-ab81fe2d4e38.png">

Reply:
Thanks for the mention. I add a little trick to fix this.